src/onNodesVisibilityChange.tsx

```diff
@@ -26,7 +26,7 @@ export const onNodesVisibilityChange = (reactflow, nodes, visible)=> {
         
         const image = node.hidden ? hiddenIcon : shownIcon
 
-        document.getElementById(node.id + "_icon").setAttribute('src', image)
+        //document.getElementById(node.id + "_icon").setAttribute('src', image)
```

Comment:
if not needed please delete

Reply:
> if not needed please delete

was commented out intentionally for further use in future when implementing the icons with hide onClick event in future